include/internal/evp.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifndef OSSL_INTERNAL_DSO_H
+# define OSSL_INTERNAL_DSO_H
+
+# include <openssl/evp.h>
+
+int evp_pkey_get_EC_KEY_curve_nid(const EVP_PKEY *pkey);
```

Comment:
Should there be some TODO comment somewhere saying this is an interim solution?

Reply:
I thought I already added such a comment???  probably somewhere else, and yeah I can duplicate it here

Reply:
Done